integration/src/integration/java/pl/allegro/tech/hermes/integration/env/HermesIntegrationEnvironment.java

```diff
@@ -85,9 +86,15 @@ public void cleanEnvironment() throws Exception {
         ArrayList<Starter<?>> reversedStarters = new ArrayList<>(STARTERS.values());
         Collections.reverse(reversedStarters);
 
-        for (Starter<?> starter : reversedStarters) {
-            starter.stop();
-        }
+        reversedStarters.stream()
+                .filter(Objects::nonNull)
```

Comment:
why are we filtering `nonNull` values? is there a possibility for a null value at all?